lib/osf-components/addon/components/institution-carousel/template.hbs

```diff
@@ -6,8 +6,8 @@
         @showIndicators={{false}}
         @prevControlLabel={{t 'general.previous'}}
         @nextControlLabel={{t 'general.next'}}
-        @prevControlIcon='fa fa-angle-left'
-        @nextControlIcon='fa fa-angle-right'
+        @prevControlIcon='far fa-angle-left'
+        @nextControlIcon='far fa-angle-right'
```

Comment:
This didn't actually fix anything. Waiting on Product to decide how we want to move forward with the institutional carousel.